Jenkinsfile

```
@@ -28,7 +28,7 @@ pipeline {
     stage('Unit Tests and CodeCoverage Test'){
       steps{
         script{
-          sh 'cp /home/ubuntu/configs/5.6.0/cat-config-test.json ./configs/config-test.json'
+          sh 'cp /home/ubuntu/configs/5.6.0/cat-config-tgdex.json ./configs/config-test.json'
```

```suggestion
          sh 'cp /home/ubuntu/configs/tgdex/cat-config-test.json ./configs/config-test.json'
```
tgdex will be a separate branch let's not group it under 5.6.0

---

Move the files in slave and update all references too

---

moved configs to tgdex directory